cli/azd/cmd/cobra_builder.go

```diff
@@ -58,28 +59,58 @@ func (cb *CobraBuilder) BuildCommand(descriptor *actions.ActionDescriptor) (*cob
 		}
 	}
 
-	// Configure action resolver for leaf commands
-	if !cmd.HasSubCommands() {
-		if err := cb.configureActionResolver(cmd, descriptor); err != nil {
-			return nil, err
-		}
+	// Configure action resolver for all commands
+	if err := cb.configureActionResolver(cmd, descriptor); err != nil {
+		return nil, err
 	}
 
 	return cmd, nil
 }
 
+func handleDocsFlag(
+	ctx context.Context, cmd *cobra.Command, container *ioc.NestedContainer) (bool, error) {
+	// Handle --docs flags for all commands
+	// Each command can use the description.options.DocumentationUrl to set what url to browse.
```

Comment:
I did!!
But then I ended up removing it, because it felt like repeating the same url for all commands!!  
And I think it is really unlikely to think about a moment when each command would have a completely different url